lib/msf/core/modules/metadata/search.rb

```diff
@@ -213,6 +214,8 @@ def is_match(params, module_metadata)
               match = [keyword, search_term] if module_metadata.stager_refname =~ regex
             when 'adapter'
               match = [keyword, search_term] if module_metadata.adapter_refname =~ regex
+            when 'session_type'
+              match = [keyword, search_term] if module_metadata.session_types != false && module_metadata.session_types.any? { |session_type| session_type =~ regex }
```

Comment:
```suggestion
              match = [keyword, search_term] if module_metadata.session_types && module_metadata.session_types.any? { |session_type| session_type =~ regex }
```
Would the above suggestion still work? 🤔